LondonBritishLibrary/orient/BLorient2082.xml

```diff
@@ -0,0 +1,375 @@
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <collection>Oriental</collection>
+                        <idno>BL Oriental 2082</idno>
+                        <altIdentifier><idno>Or. 2082</idno></altIdentifier>
+                        <altIdentifier><idno>Strelcyn 54</idno></altIdentifier>
+                    </msIdentifier>
+                    <msContents><summary/></msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">175</measure>
+                                </extent>
+                                <condition key="good"><locus target="#175"/> is only partly preserved (a strip about 120 mm. wide). Between
+                                    <locus target="#159"/> and <locus target="#160"/> three leaves (blank?) have been cut out.</condition>
+                            </supportDesc> 
+                        </objectDesc> 
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <locus target="#1r"/>
+                                    <desc type="findingAid">In the upper margin a note by <persName ref="PRS14422FMayer">F. Mayer</persName>
+                                        indicating the contents of the first part.</desc> 
+                                    <q xml:lang="de"><foreign xml:lang="gez">ስንክሳር፡</foreign> - Senkesar, zweite Hälfte beginnend mit März bis zum
+                                        Ende des abessinischen Jahres</q>
+                                </item>
+                                <item xml:id="e2">
+                                    <locus target="#160r"/>
+                                    <desc type="findingAid">A note by <persName ref="PRS14422FMayer">F. Mayer</persName>
+                                        indicating the contents of the second part.</desc> 
+                                    <q xml:lang="de"><foreign xml:lang="gez">ባሕረ፡ ሐሳብ፡</foreign> - Senkesar, zweite Hälfte beginnend mit März bis zum
+                                        Ende des abessinischen Jahres</q>
+                                </item>
+                                <item xml:id="e3">
+                                    <locus target="#1r"/>
+                                    <desc type="AcquisitionNote"/>
+                                    <q xml:lang="en">Presented by <persName role="bequeather" ref="PRS0000000000000000000000000000">
```

Comment:
?

Reply:
I forgot to create this PRS ID. I will do in a minute.